src/libraries/System.Runtime.InteropServices.JavaScript/tests/JSImportGenerator.UnitTest/Compiles.cs

```diff
@@ -67,6 +67,7 @@ public async Task ValidateGeneratedSourceOutput_AllAnnotatedParameters()
                             unsafe partial class Basic
                             {
                                 [global::System.Diagnostics.DebuggerNonUserCode]
+                                [global::System.Runtime.Versioning.SupportedOSPlatformAttribute("browser")]
```

Comment:
Note that this is a kind of a breaking change (new warnings can appear in existing code), not sure if it needs to be announced/documented somewhere?